model/gen-ai/deprecated/registry-deprecated.yaml

```diff
@@ -129,8 +135,8 @@ groups:
               value: "xai"
               brief: 'xAI'
               deprecated:
-                reason: renamed
-                renamed_to: "x_ai"
+                reason: uncategorized
+                note: Use `x_ai` on the `gen_ai.provider.name` attribute
```

Comment:
this one feels a little odd, is there no replacement in this (deprecated) enum? would it make sense to "undeprecate" this value (within the still deprecated enum)?